### PR TITLE
Card, Navigation and Toast

### PR DESCRIPTION
**Pull request by Chiragmodi01**
image
image
image

### CODE REVIEW

File: Docsite/Docsite Comp CSS/Card.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    position: absolute;
+    top: 50%;
+    border: 1px solid black;
+    height: 1.5rem;
+    width: 1.5rem;
+    background-color: #f3f3f3c7;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 100%;
```
Comment (aurobindoGupta) [inline Docsite/Docsite Comp CSS/Card.css:26]: Does this work? please check

File: Docsite/Docsite Comp CSS/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+.header {
+    background-color: white;
+}
+.header_top-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: .5rem 1.5rem;
+    border-bottom: 1px solid #e9e9e9;
```
Comment (aurobindoGupta) [inline Docsite/Docsite Comp CSS/Navigation.css:16]: use var()

File: Docsite/Docsite Comp CSS/Navigation.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+/* header main */
+.header_main-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 1.5rem;
+    border-bottom: 1px solid #e9e9e9;
```
Comment (aurobindoGupta) [inline Docsite/Docsite Comp CSS/Navigation.css:40]: same as  above

File: Docsite/Docsite Comp CSS/Navigation.css [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    align-items: center;
+    justify-content: center;
+}
+.header_main_nav-search-box {
+    outline: none;
+    border: none;
+    padding: .4rem;
+    width: 5rem;
+    font-size: 1rem;
+    color: black;
```
Comment (aurobindoGupta) [inline Docsite/Docsite Comp CSS/Navigation.css:75]: variable

**Review by aurobindoGupta — APPROVED** [review]
nice work

**Review by harshitpaliwal95 — APPROVED** [review]
- looks good

**Pull request merged**